bundles/org.openhab.binding.mqtt.awtrixlight/README.md

```diff
@@ -283,11 +283,17 @@ The following actions are supported:
 
 ```java
 Bridge mqtt:broker:myBroker [ host="localhost", port=1883 ]
-Bridge mqtt:awtrix-clock:myBroker:myAwtrix "Living Room Display" (mqtt:broker:myBroker) [ basetopic="awtrix", appLockTimeout=10, lowBatteryThreshold=25 ] {
-    Thing awtrix-app clock "Clock App" [ appname="clock", useButtons=true ]
-    Thing awtrix-app weather "Weather App" [ appname="weather" ]
-    Thing awtrix-app calendar "Calendar App" [ appname="calendar" ]
-    Thing awtrix-app custom "Custom App" [ appname="custom" ]
```

Comment:
Above gives the impression, the `awtrix-clock` bridge is a sibling to the `broker`.
It's actually meant to be a child